callstack/react-native-builder-bob
# chore(bob)!: remove AAR target support

### Summary

This removes the (non-working) support for `aar` targets.
Originally, the `aar` support was added with #4 to support AndroidX. However, it's not needed anymore so we are removing the support.

### Test plan

1. Create a new library with `npx create-react-native-library`
2. Add `aar` to targets
3. run `yarn prepack`
4. Make sure an error is printed and the program stops.
